ui/ui-components/components/model/overview/EntityInfoHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import Link from "next/link";
+import EnterpriseLink from "../../GrouparooLink";
+import { formatName } from "../../../utils/formatName";
+import StateBadge from "../../badges/StateBadge";
+
+type Entity = Parameters<typeof formatName>[0];
```

Comment:
Why not just export this type from `formatName`?

Reply:
Fixed